### ISSUE TITLE
Allow to override short option

### ISSUE THREAD

**Issue by georgiosd** [natemcmaster/CommandLineUtils#17]
Awesome work!

I have `Delay1` and `Delay2` options, which obviously clash in "-d"
Would be great if I could specify just the short override to "-d1" and "-d2"!

**Comment by natemcmaster** [comment]
In 2.1.0-beta, you can override the default by passing in a value for template to OptionAttribute.
```c#
[Option("-d1|--detail1 <DETAIL1>")]
public string Detail1 { get; set; }
```

Or, once PR #18 is merged, you should be able to shorten this to
```c#
[Option(ShortName = "d1")]
public string Detail1 { get; set; }
```

**Comment by georgiosd** [comment]
Thank you sir.
